routing/router.go

```diff
@@ -26,8 +27,16 @@ func NewRouter(conf *config.Config) *mux.Router {
 	// the webapp js files and other static content. Then tell the router about our fixed
 	// routes which pass all static file requests to the file handler.
 	fileServerHandler := http.FileServer(http.Dir(conf.Server.StaticContentRootDirectory))
-	router.PathPrefix("/console").Handler(http.StripPrefix("/console", fileServerHandler))
-	router.PathPrefix("/").Handler(fileServerHandler)
+	router.PathPrefix("/static").Handler(fileServerHandler)
+	router.PathPrefix("/asset-manifest.json").Handler(fileServerHandler)
+	router.PathPrefix("/favicon.ico").Handler(fileServerHandler)
+	router.PathPrefix("/manifest.json").Handler(fileServerHandler)
+	router.PathPrefix("/service-worker.js").Handler(fileServerHandler)
```

Comment:
Note here that we could have another strategy: instead of listing all files here (these are actually root files that cannot be put in "static" folder), we may change our client-side routes to add a disambiguation prefix, such as:
`/r/namespaces/etc.`

So that the rules here would be simplified:

```
	router.PathPrefix("/r/").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		http.ServeFile(w, r, conf.Server.StaticContentRootDirectory+"/index.html")
	})
        router.PathPrefix("/").Handler(fileServerHandler)
```

Any preference?

Reply:
What changed? Because I had this working without having to add all the individual files. There were two routes we needed to proxy to the file server - "/" and "/static" IIRC.

Reply:
nothing changed on the frontend, it was already not entirely working.
Again, I'm speaking about URLs like "http://sws/namespace/myproject"

They would fall under the fileserver handler, look for a file name "/namespace/myproject"... and return 404 because that file doesn't exist. It must be forwarded to index.html

Reply:
PS: if you want to reproduce the bug, just open the console, navigate to services (for instance) and hit F5. Gives 404.